Nexpo/Controllers/EventsController.cs

```diff
@@ -90,6 +90,36 @@ public async Task<ActionResult> GetTicketsForEvent(int id)
             }
             return Ok(namedTickets);
         }
+
+        /// <summary>
+        /// Returns all food preferences of event. 
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{id}/foodprefs")]
+        [Authorize(Roles = nameof(Role.CompanyRepresentative))]
```

Comment:
Is this correct? Shouldn´t it only be accessible for admin since ARKAD is responsible for providing the food for the events? 
If we want companyReps to access this data we need to add a check that ensures that their part of the company that hosts the event, but since the host attribute currently is just a string this may introduce some problems. 
Currently is the host of the event only a string due to that they necessarily not a company at the fair, s